src/main/resources/archetype-resources/src/test/java/PearPackageIT.java

```diff
@@ -22,39 +39,34 @@
  */
 public class PearPackageIT {
 
-	/*
-	 * Make sure that the annotator PEAR package 
-	 * can be installed by the UIMA PackageInstaller
-	 */
-	@Test
-	public void testInstallPearPackage() {
-		
-		this.installPackage();			
-	}
-	
-	/*
-	 * Make sure that the annotator is able to 
-	 * process text without exceptions
-	 */
 	@Test
-	public void testProcessText() throws IOException, UIMAException {
+	public void testProcessText() throws Exception {
 		
-		PackageBrowser packageBrowser = this.installPackage();		
+		PackageBrowser packageBrowser = this.installPackage();
 		XMLInputSource xmlInputSource = new XMLInputSource(packageBrowser.getComponentPearDescPath());
 		PearSpecifier pearSpecifier = UIMAFramework.getXMLParser().parsePearSpecifier(xmlInputSource);
 		AnalysisEngine analysisEngine = UIMAFramework.produceAnalysisEngine(pearSpecifier, new SimpleNamedResourceManager(), null);
 		
-		JCas jCas = JCasFactory.createJCas();
-		jCas.setDocumentText("Test document text");
-		
-		analysisEngine.process(jCas);
+		JCas jcas = JCasFactory.createJCas();
+		CAS cas = jcas.getCas();
```

Comment:
Just work with the JCas? Or if that does not work, then just work with the CAS, but having both looks overcomplicated.

```
CAS cas = CasFactory.createText("Test document text");
```